parser/gen/CEL.g4

```diff
@@ -183,8 +180,6 @@ STRING
   | RAW '\'\'\'' .*? '\'\'\''
   ;
 
-fragment RAW : 'r' | 'R';
-
 BYTES : ('b' | 'B') STRING;
 
 IDENTIFIER : (LETTER | '_') ( LETTER | DIGIT | '_')*;
```

Comment:
No newline at end of file.  (Fix your IDE! :-)

Reply:
Done. :)